.github/workflows/staging.yml

```diff
@@ -115,6 +115,19 @@ jobs:
           SLACK_MESSAGE: Deploying to ${{ matrix.url }} has *FAILED!*
           SLACK_CHANNEL: yjdh-alerts
         if: failure()
+
+      - name: Deploy Hourly Cronjobs
+        uses: City-of-Helsinki/setup-cronjob-action@main
+        with:
+          image_repository: ghcr.io/city-of-helsinki/${{ github.event.repository.name }}
```

Comment:
It's recommended to give a name to a cronjob, as it's become mandatory later when we have multiple jobs in one namespace. https://github.com/City-of-Helsinki/setup-cronjob-action